reference/changelog.md

```diff
@@ -58,6 +63,102 @@ redirect_from: /install/changelog.html
+
+#### `pulumi refresh`
+
+We've made a bunch of improvements in `pulumi refresh`. Some of these improve the UI during a refresh (for example, clarifying text about the underyling operations) as well fixing bugs with refreshing certain types of objects (for example CloudFront CDNs).
+
+#### `pulumi up` and `pulumi new`
+
+You can now pass a URL to a Git repository to `pulumi up <url>` to deploy a project without having to manage its source code locally. This works like `pulumi new <url>`, but configures and deploys the project from a temporary directory that will be cleaned up automatically after the update.
+
+`pulumi new` now outputs an error when the current working directory (or directory specified explicitly via the `--dir` flag) is not empty. Additionally, `pulumi new` now runs a preview of an initial update at the end of its operation and asks if you would like to perform the update.
+
+Both `pulumi up` and `pulumi new` now support `-c` flags for specifying config values as arguments (e.g. `pulumi up <url> -c aws:region=us-east-1`).
+
+#### Miscellaneous Improvements
+
+In addition to the above features, we've made a handfull of day to day improvements in the CLI:
+
+- Support `pulumi` in a projects in a Yarn workspaces. `[pulumi/pulumi#1893](https://github.com/pulumi/pulumi/pull/1893)`
+- Improve error message when there are errors decrypting secret configuration values. `[pulumi/pulumi#1815](https://github.com/pulumi/pulumi/pull/1815)`
+- Don't fail `pulumi update` when plugin discovery fails. `[pulumi/pulumi#1745](https://github.com/pulumi/pulumi/pull/1745)`
+
+### @pulumi/pulumi 0.15.1
+
+- New helpers for extracting and validating values from `pulumi.Config`. `[pulumi/pulumi#1843](https://github.com/pulumi/pulumi/pull/1843)`
+- Support serializng "factory" functions. `[pulumi/pulumi#1804](https://github.com/pulumi/pulumi/pull/1804)`
+
+### @pulumi/aws 0.15.1
+
+- Add dotnetcore2.1 as a runtime for AWS lambdas (thanks to **[@Jtango18](https://github.com/Jtango18)**!). `[pulumi/pulumi-aws#302](https://github.com/pulumi/pulumi-aws/pull/302)`
+- Expose 'factory functions' from pulumi-aws. `[pulumi/pulumi-aws#304](https://github.com/pulumi/pulumi-aws/pull/304)`
+- Use stronger IAM doc types. `[pulumi/pulumi-aws#311](https://github.com/pulumi/pulumi-aws/pull/311)`
+- Auto-name aws.elasticache.Cluster#clusterId, `[pulumi/pulumi-aws#313](https://github.com/pulumi/pulumi-aws/pull/313)`
+
+### @pulumi/aws-infra 0.15.1
+
+- Update dependencies versions for `@pulumi/pulumi` and `@pulumi/aws`.
+
+### @pulumi/aws-serverless 0.15.1
+
+- Update dependencies versions for `@pulumi/pulumi` and `@pulumi/aws`.
```

Comment:
Should we go ahead and publish `0.15.2` and pull in the `Queue` contribution before publishing these notes?